modules/osd-create-cluster-ccs.adoc

```diff
@@ -198,6 +201,26 @@ You must ensure that your VPC is configured with a public and a private subnet f
 ====
 endif::osd-on-aws[]
 
+. If you opted to configure a cluster-wide proxy, provide your proxy configuration details on the *Cluster-wide proxy* page:
++
+--
+.. Complete at least one of the following fields:
```

Comment:
Please refer to the comment above regarding the ISG's guidance on using _enter_ instead of _complete_.

Reply:
Thank you for highlighting this. Please see my reply to the other thread for details. I have decided on "Enter a value in at least one of the following fields:".